src/soundfile.js

```diff
@@ -221,9 +221,11 @@ define(function (require) {
       request.onload = function() {
         if (request.status === 200) {
           // on sucess loading file:
+          if (!self.panner) return;
```

Comment:
This does the trick, but I wonder if there's a better way to check if the soundfile has been disposed. Or, better yet, to cancel any incomplete callbacks when the soundfile is disposed.